bmi_calculator/pubspec.yaml

```diff
@@ -0,0 +1,26 @@
+name: bmi_calculator
+description: A new Flutter module project.
+
+version: 1.0.0+1
+
+environment:
+  sdk: ">=2.17.6 <3.0.0"
+
+dependencies:
+  flutter:
+    sdk: flutter
+
+  cupertino_icons: ^1.0.2
+
+dev_dependencies:
+  flutter_test:
+    sdk: flutter
+  flutter_lints: ^2.0.0
+
+flutter:
+  uses-material-design: true
+
+  module:
+    androidX: true
+    androidPackage: com.example.bmi_calculator
+    iosBundleIdentifier: com.example.bmiCalculator
```

Comment:
Refactor package name to follow standard as set in this repo. [Mentioned in readme](https://github.com/nisrulz/flutter-examples#pull-requests)